corehq/toggles.py

```diff
@@ -265,6 +265,43 @@ def enabled(self, item, namespace=Ellipsis):
             or super(PredictablyRandomToggle, self).enabled(item, namespace)
         )
 
+
+class DynamicallyPredictablyRandomToggle(PredictablyRandomToggle):
+    """
+    A PredictablyRandomToggle whose randomness can be configured via the database/UI.
```

Comment:
There's no UI component at the moment though right?

Reply:
correct, going to see how hard it is to add. we could also just manage via CLI if we wanted to really restrict it to developers.